### DISCUSSION TITLE
Collab session should not require a voice call

### DISCUSSION THREAD

**Discussion by smpallen99** [zed-industries/zed#24048] [Feature Requests]
### Summary

After tying my first collab session today, I had issue with the call #9231 so I needed to use something else for the voice. I could not find a way to start a collab session without a having a call. 

I can think of a number of scenarios where I would like to use something else for voice (even a video call), or perhaps, only use chat. So I think the voice call should be optional. 

As further feedback...

First, I found the features valuable and will probably use zed for pairing (don't do that much though), training, or just walking through some code with someone else. I tried the screen sharing too and it worked fine.

And for the potential improvements:

* I found the UX difficult to undestand. It took a while of "playing" to undestand the relationship between all the items in the collab panel, the need to invite the other person to the channel I created even after establishing the (broken) call. 
* the participant icons across the header was nice, especially the speech detection indication (very nice touch). However, I was expecting to be able to interact with those icons. 
* a way to troubleshoot speech issues will probably be needed. I develop voice and video WebRTC integration on our messaging product, so I have some experience in the topic. Speech path can be a tricky to troubleshoot.
* Regarding screen sharing, the first thing I wanted to test was to share only my browser along with the editor's code. In this case, sharing my complete screen was infeasable since I'm running full 4K and the other end was using a much lower resolution. 

However, even with my feedback, I think the collab feature is well done and valuable. Good start for a beta and room for improvement :)

**Comment by ConradIrwin** [comment]
Related ideas from @Otterpocket:
![image](https://github.com/zed-industries/zed/assets/94272/1c1c456a-2e1e-4907-913c-8696ba377b5f)

<blockquote>

This functionality also means that people who would like to voice chat have to explicitly choose to do so, meaning they can just be in a channel listening to music, suggesting things and pairing on code, using text chat etc. When stuff gets to the point of needing voice chat, it can just be initialised, users in the channel are notified that an audio session has started and have the option to join.

I like the idea of audio sessions only being active when there are at least 1 person in them.

When it comes to inviting people, I think you should be able to right click the audio channel and select from.

------------------------------------------------
Invite all users in channel to audio session

------Users in channel-------
-- Ben
-- Bob
-- Bill

---------- All users  ------------
-- Jim
-- James
-- John
</blockquote>

**Comment by probably-neb** [comment]
Seeing as there are some interesting ideas here, I'm going to move this to discussions to allow for referencing and further discussion as part of our transition to using discussions for feature ideas & feedback.
